administrator/language/en-GB/en-GB.plg_system_shortcut.ini

```diff
@@ -0,0 +1,61 @@
+PLG_SYSTEM_SHORTCUT="System - Keyboard Shortcut"
+PLG_SYSTEM_SHORTCUT_XML_DESCRIPTION="System - Keyboard Shortcut Plugin"
+PLG_SYSTEM_SHORTCUT_TRUE = "True"
+PLG_SYSTEM_SHORTCUT_FALSE = "False"
+PLG_SYSTEM_SHORTCUT_HASSHIFT_LBL = "Has Shift"
+PLG_SYSTEM_SHORTCUT_HASALT_LBL = "Has Alt"
+PLG_SYSTEM_SHORTCUT_HASCONTROL_LBL = "Has Control"
+PLG_SYSTEM_SHORTCUT_BUTTONS_LBL = "Buttons"
+PLG_SYSTEM_SHORTCUT_EDITOR_LBL = "Editor"
+PLG_SYSTEM_SHORTCUT_BUTTON_NEW_LBL = "Button New"
+PLG_SYSTEM_SHORTCUT_BUTTON_NEW_KEY_LBL = "Key"
+PLG_SYSTEM_SHORTCUT_BUTTON_NEW_KEY_DESC = ""
```

Comment:
remove all empty _DESC entries if you don't use tem, Also since j4 we use DESCRIPTION ( see second line)